packages/terra-application-utility/src/utility/_UtilityMenu.scss

```diff
@@ -0,0 +1,135 @@
+:local {
+  .header-utility-menu {
+    background-color: var(--terra-application-header-utility-menu-background-color);
+    font-family: inherit;
+    width: 100%;
+    /* stylelint-disable selector-max-compound-selectors*/
+    .header {
+      display: flex;
+      flex-direction: column;
+      flex-shrink: 0;
+      font-size: var(--terra-application-header-utility-menu-header-font-size, 1rem);
+      font-weight: var(--terra-application-header-utility-menu-header-font-weight, normal);
+      line-height: var(--terra-application-header-utility-menu-header-line-height, 1.2);
+      margin-top: var(--terra-application-header-utility-header-margin-top, 0.7143rem);
+
+      /* stylelint-disable max-nesting-depth */
+      .content-container {
+        display: flex;
+        justify-content: space-between;
+        margin-bottom: var(--terra-application-header-utility-header-margin-bottom, 0.7143rem);
+
+        .left-content-container {
+          align-items: center;
+          display: flex;
+          justify-content: flex-start;
+
+          .initial-page-header-text {
+            display: inline-block;
+            margin-left: var(--terra-application-header-utility-menu-initial-page-header-text-margin-left, 0.7143rem);
+          }
+
+          .noninitial-page-header-text {
+            margin-left: var(--terra-application-header-utility-menu-noninitial-page-header-text-margin-left, 0);
+          }
+
+          .icon-left {
+            flex-shrink: 0;
+            width: var(--terra-application-header-utility-menu-chevron-width, 0.7143rem);
+          }
+        }
+      }
+    }
+
+    .right-content-container {
+      display: flex;
+      justify-content: flex-end;
+
+      .icon-close {
+        flex-shrink: 0;
+        width: var(--terra-application-header-utility-menu-page-chevron-width, 1rem);
+      }
+      //   /* stylelint-enable max-nesting-depth */
+    }
+
+    .body {
+      list-style-type: none;
+      margin: 0;
+      overflow: hidden;
+      padding: 0;
+    }
+
+    .footer {
+      display: flex;
+      flex-direction: column;
+      flex-shrink: 0;
+      /* stylelint-enable selector-max-compound-selectors*/
+    }
+  }
+
+  .menu-utility-menu {
```

Comment:
`.header-utility-menu` and `.menu-utility-menu` and their nested selectors share a lot of styles. I'd recommend using a `.utility-menu` selector to group shared styles under and use `.header-utility-menu` and `.menu-utility-menu` for unique styles. This will reduce the size of CSS for this component.

Reply:
Flattened [them](https://github.com/cerner/terra-framework/pull/103/commits/efd64c3eed5f7146aaefa0505339e34e92f75bed#diff-a67eb75e7de351f9d43b82df1f4b5f13R2).